contact_form_mail/settings.py

```diff
@@ -73,18 +73,24 @@
 # Database
 # https://docs.djangoproject.com/en/3.2/ref/settings/#databases
 
-
 DATABASES = {
     'default': {
-        'ENGINE': 'django.db.backends.postgresql',
-        'NAME': os.environ.get('DB_NAME'),
-        'HOST': os.environ.get('DB_HOST', 'localhost'),
-        'USER': os.environ.get('DB_USER', 'postgres'),
-        'PASSWORD': os.environ.get('DB_PASSWORD', ''),
-        'PORT': os.environ.get('DB_PORT', '5432'),
+        'ENGINE': 'django.db.backends.sqlite3',
```

Comment:
revert these changes. Also you can just create a postgres database.

Reply:
sorry i dont understand what you mean

Reply:
by revert I mean to undo the change. I see you are attempting to use an sql lite database. However, I want the app along with its functionality to be built and tested with Postgres. So uncomment the lines below starting with line 83.